src/status_im/subs/multiaccount.cljs

```diff
@@ -207,14 +207,13 @@
  (fn [multiaccount]
    (pos? (count (get multiaccount :images)))))
 
-(defn- replace-multiaccount-image-uri [multiaccount port]
+(defn- replace-multiaccount-image-uri [{:keys [key-uid] :as multiaccount} port]
   (let [public-key (:public-key multiaccount)
         identicon (image-server/get-identicons-uri port public-key)
         multiaccount (assoc multiaccount :identicon identicon)
         images (:images multiaccount)
         images (reduce (fn [acc current]
-                         (let [key-uid (:keyUid current)
-                               image-name (:type current)
+                         (let [image-name (:type current)
```

Comment:
What's the logic behind this? I saw there are two images (same image with different size) with same `:keyUid` in multiaccount `:images`. Is it possible that there will be images with different `:keyUid`?

Reply:
for the old one `key-uid (:keyUid current)`, key-uid may be nil in some cases, i guess there must be some issues with `status-go` that it forget to return value of key-uid which will cause CASE 2 reported from #14236 . the same image with different type(thumbnail/large) share the same :keyUid

Reply:
Then we better raise this in status-go channel or repo, since this might affect somewhere else as well.

Reply:
i can raise an issue for this later, but it's another issue, can we solve this issue first? :)

Reply:
I don't think it's another issue, this makes finding the root cause harder.

> the same image with different type(thumbnail/large) share the same :keyUid

If this is the case, won't this make `reduce` generate some invalid images URLs if there are more than one set of image in multiaccount `images`?

Reply:
no, they r valid, the url of images differed by imageName.

Reply:
i mean status-go didn't return value of key-uid is another issue

Reply:
> no, they r valid, the url of images differed by imageName.

If same image with different size share the same `key-uid`, won't there will be URL with invalid `&keyUid=` once there are multiple images?

https://github.com/status-im/status-mobile/blob/09675826393928d44c5a41b41a6b17727f5f703b/src/status_im/utils/image_server.cljs#L21